.github/bin/release-plan/npm-prepublish-scripts.mjs

```diff
@@ -0,0 +1,35 @@
+import { spawn } from 'child_process';
+import { platform } from 'process';
+
+export async function npmPrepublishScripts(packageDirectory, packageName) {
```

Comment:
After deleting `scripts` from `package.json` the pre publish scripts will be gone.

So we need to run these manually before we minify `package.json` for publishing.